spec/features/system/admin/instance/user_management_spec.rb

```diff
@@ -22,22 +22,17 @@
         expect(page.all('tr.instance_user').count).to eq(instance.instance_users.count)
       end
 
-      scenario 'I can change a user\'s role' do
+      scenario "I can change a user's role", js: true do
         visit admin_instance_users_path
 
         user_to_change = instance_users.sample
-        field = find('tr.instance_user td', text: user_to_change.user.email)
-
-        within field.find(:xpath, '..') do
-          select '', from: 'instance_user_role'
-          find_button('update').click
+        within find(content_tag_selector(user_to_change)) do
```

Comment:
@kxmbrian Sorry, just noticed this. The original spec actually tested both failure and success case, the new one you wrote only tested the success path, is that intended ?